### ISSUE TITLE
Lag in work days?

### ISSUE THREAD

**Issue by frejon** [adamaltmejd/covid#3]
Alternative title"why Tuesdays don't have any reported deaths with a 5-6 day lag". It's because that would be reports during Saturdays and Sundays. Almost no reports during those days. Also true for bank holidays. A similar pattern but for a different group is there also for Wednesdays.

Tegnell himself said that the data is more or less correct after "4 arbetsdagar". So it'd be nice feature if you could make the graphs after that instead.

**Comment by adamaltmejd** [comment]
Thanks and you very right but I but don't understand what you are after? I mean its obviously not true that the graphs are "finished" after 4 days. Blue and dark yellow are added > 4 days. I'll close this but feel free to reopen if you have a specific issue with the graph that you would like me to change.

**Comment by frejon** [comment]
It'd be nice if the data was grouped in number of workdays (lag) instead of days.

**Comment by frejon** [comment]
Do you see the difference? I can't easily tell if Tegnells statement is true because both blue and dark yellow can be less or equal to 4 WORKdays. En normal vecka har 7 dagar och 5 arbetsdagar...

**Comment by adamaltmejd** [comment]
Ohh, I see. Thats an interesting idea! And then just adding the deaths added during weekends to the closest (lower?) workday-group?

**Comment by frejon** [comment]
yeah, that would work! It'd be the group "reported in 1 workday or less". Or whatever you want to call it.

**Comment by EliasHasle** [comment]
I think Iagree. This could eliminate one source of systematic error. Another, possibly better, solution, would be to model the dependence on day of week. 

Another problem, harder to solve, is that lag can depend on workload, namely such that when there are fewer deaths, the lags tend to be shorter. Maybe this can be compensated for by using weighted averages of lags, with weights decaying over time. Except that would stil be very laggy, and worst when a second wave suddenly explodes. Maybe one can have a _very_ simple prediction model for the dependency of lags on actual number of deaths, in the form of an average multiplier modelled as a strictly increasing function of number of deaths, and harmonize the shaping parameters of that function with both the history and the resulting nowcasts?

Yet another problem is summer vacation (+exhaustion), which may be expected to increase the lags again.
